packages/patternfly-4/content/design-guidelines/content/grammarandterminology.md

```diff
@@ -73,6 +64,63 @@ Use sentence-style capitalization for all text elements, **except:**
 
 Sentence style capitalizes only the first word of each sentence and proper nouns, like names.
 
+## Date and time
+
+Whenever possible, it is best to enable users to customize the format in which date and time are displayed in the interface.
+
+These date and time standards are based in American English, but date and time is formatted differently depending on language.
+
+### Date
+Use the default US style of **Month DD, YYYY**. Use **DD Month YYYY** for non-US locations.
+
+Whenever possible, write out the full name of the month. If you have space constraints, use the 3 letter abbreviation for the name of the month.
+
+Jan
```

Comment:
I think you intend line breaks here and below but they are not showing in the GitHub preview.